devel/external/test_photon_vs_fft/cosmos_sersics_sample_N300.asc

```diff
@@ -0,0 +1,301 @@
+# ident  n_sersic  half_light_radius [arcsec] |g|
```

Comment:
I don't recall deliberately doing this. My guess is that my git process might have done this while changing branches at some point.